content/en/docs/concepts/services-networking/_index.md

```diff
@@ -1,63 +1,175 @@
 ---
 title: "Services, Load Balancing, and Networking"
 weight: 60
+simple_list: true
 description: >
   Concepts and resources behind networking in Kubernetes.
 ---
 
-## The Kubernetes network model
+The [Kubernetes network model](#kubernetes-network-model) enables container networking within a pod and between pods
+on the same or different {{< glossary_tooltip text="nodes" term_id="node" >}}.
 
-Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address. 
-This means you do not need to explicitly create links between `Pods` and you 
-almost never need to deal with mapping container ports to host ports.  
-This creates a clean, backwards-compatible model where `Pods` can be treated 
-much like VMs or physical hosts from the perspectives of port allocation, 
-naming, service discovery, [load balancing](/docs/concepts/services-networking/ingress/#load-balancing), 
-application configuration, and migration.
+Kubernetes networking addresses four concerns:
+- Containers within a Pod [use networking to communicate](/docs/concepts/services-networking/dns-pod-service/) via loopback.
+- Cluster networking provides communication between different Pods.
+- The [Service](/docs/concepts/services-networking/service/) API lets you
+  [expose an application running in Pods](/docs/tutorials/services/connect-applications-service/)
+  to be reachable from outside your cluster.
+  - [Ingress](/docs/concepts/services-networking/ingress/) and [Gateway](https://gateway-api.sigs.k8s.io/) provide
+    extra functionality specifically for exposing your applications, websites and APIs, usually to clients outside
+    the cluster. Ingress and Gateway often use a load balancer to make that work reliably and at scale.
+- You can also use Services to
+  [publish services only for consumption inside your cluster](/docs/concepts/services-networking/service-traffic-policy/).
+
+The [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) tutorial lets you learn
+about Services and Kubernetes networking with a hands-on example.
+
+Read on to learn more about the [Kubernetes network model](#kubernetes-network-model).
+
+## Terminology
+
+L2 bridge
+: a (virtual) [layer 2](https://en.wikipedia.org/wiki/Data_link_layer) bridge enabling inter-pod connectivity on the same host.
```

Comment:
Have a look at https://github.com/kubernetes/website/pull/39890 and existing reviews (maybe you did already). I'd like to talk as little as possible about encapsulation; maybe it's OK to yank any mention?

Reply:
In #39890, the diagrams were introduced with phrases like "in this example", whereas that wording was lost in this PR. I think that's the problem. These diagrams do not show "the Kubernetes network model". They show one particular concrete _implementation_ of the abstract Kubernetes network model.

If you want to fully explain the diagram to the reader, you may need to talk about L2 bridges and such. But your explanation is just about _the diagram_ (or about what this particular unnamed plugin is doing), not about "Kubernetes networking" in general. Many network plugins use an L3 bridge instead of an L2 bridge, and some don't use a bridge at all and have an architecture which looks nothing at all like these diagrams. (eg, if you use the `amazon-vpc-cni-k8s` or `azure-container-networking` plugins then each pod would be directly connected to the same network as the node itself is).